doc/source/whatsnew/v0.22.0.txt

```diff
@@ -294,6 +294,7 @@ Indexing
 - Bug in tz-aware :class:`DatetimeIndex` where addition/subtraction with a :class:`TimedeltaIndex` or array with ``dtype='timedelta64[ns]'`` was incorrect (:issue:`17558`)
 - :func:`Index.to_series` now accepts ``index`` and ``name`` kwargs (:issue:`18699`)
 - :func:`DatetimeIndex.to_series` now accepts ``index`` and ``name`` kwargs (:issue:`18699`)
+- Bug in :class:`DatetimeIndex.insert` where inserting ``pd.NaT`` into a timezone-aware index incorrectly raised (:issue:`16357`)
```

Comment:
I think `:class:` should be `:func:` or `:meth:` instead, and it looks like previous convention has been to just use <code>\`\`NaT\`\`</code> with the `pd.` omitted.